js/data.js

```diff
@@ -1,122 +1,86 @@
 const crafting = {
-    heatsink: [{
-        quantity: 2,
-        material: "Basic Conductors"
-    },
-    {
-        quantity: 2,
-        material: "Heat Conduction Wiring"
-    }],
-    flak: [{
-        quantity: 2,
-        material: "Sulphur"
-    },
-    {
-        quantity: 3,
-        material: "Carbon"
-    },
-    {
-        quantity: 4,
-        material: "Nickel"
-    }],
-    gauss: [{
-        quantity: 2,
-        material: "Focus Crystals"
-    },
-    {
-        quantity: 2,
-        material: "Guardian Power Conduit"
-    },
-    {
-        quantity: 3,
-        material: "Manganese"
-    },
-    {
-        quantity: 4,
-        material: "Guardian Wreckage Components"
-    }],
-    gaussPrem: [{
-        quantity: 6,
-        material: "Filament Composites"
-    },
-    {
-        quantity: 6,
-        material: "Guardian Technology Components"
-    },
-    {
-        quantity: 8,
-        material: "Manganese"
-    },
-    {
-        quantity: 10,
-        material: "Focus Crystals"
-    }],
-    shard: [{
-        quantity: 3,
-        material: "Carbon"
-    },
-    {
-        quantity: 2,
-        material: "Vanadium"
-    },
-    {
-        quantity: 3,
-        material: "Crystal Shards"
-    },
-    {
-        quantity: 3,
-        material: "Guardian Power Cell"
-    },
-    {
-        quantity: 5,
-        material: "Guardian Wreckage Components"
-    }],
-    shardPrem: [{
-        quantity: 8,
-        material: "Carbon"
-    },
-    {
-        quantity: 4,
-        material: "Vanadium"
-    },
-    {
-        quantity: 8,
-        material: "Crystal Shards"
-    },
-    {
-        quantity: 6,
-        material: "Guardian Power Cell"
-    }],
-    limpets: [{
-        quantity: 10,
-        material: "Iron"
-    },
-    {
-        quantity: 10,
-        material: "Nickel"
-    }],
-    afmu: [{
-        quantity: 2,
-        material: "Nickel"
-    },
-    {
-        quantity: 2,
-        material: "Zinc"
-    },
-    {
-        quantity: 2,
-        material: "Chromium"
-    },
-    {
-        quantity: 3,
-        material: "Vanadium"
-    }],
-    lifeSupport: [{
-        quantity: 2,
-        material: "Iron"
-    },
-    {
-        quantity: 1,
-        material: "Nickel"
-    }]
+    heatsink: {
+        max: 150, // max synths per item due to material capacity limit

```

Comment:
It isn't entirely great form to include comments in objects, but javascript luckily doesn't care (Also, is this comment really needed? I feel like the field name kinda indicates what it is)

Reply:
fixed